piccolo_api/crud/endpoints.py

```diff
@@ -423,14 +423,32 @@ async def get_ids(self, request: Request) -> Response:
 
         values = await query.run()
 
+        # if value_type not in or UUID
+        try:
+            target_pk_type = [
+                i._meta.params["target_column"].value_type not in (int, uuid)
+                for i in self.table._meta._foreign_key_references
+            ][0]
```

Comment:
I see the idea. I think the problem though is it's having to do `[0]`. If there the `_foreign_key_references` contains more than 1 item, we're guessing that the first one is correct.

I think we're going to have to add a GET parameter so the Piccolo Admin UI can explicitly tell us which column it's interested in.

Reply:
Yes. That make sense. I will try to add a GET parameter.

Reply:
Cool, thanks. I think another advantage of the GET param is if someone doesn't pass it we can return the same response as before, so there are no breaking API changes.

Reply:
I didn't get far, but we can pass `target_column` as a query param 
```python
target_column = request.query_params.get("target_column")
    if target_column is not None:
        return JSONResponse({i["readable"]: i["readable"] for i in values})
```
and return the result like this
```
# ./api/tables/movie/ids/?target_column=
{
   'Star Wars': 'Star Wars',
   'Return of the Jedi: 'Return of the Jedi,
   ...
}
```
(we don't need to pass anything to query param and return all results because the `KeySearchModal` already needs all results) 
or without the `target_column` return standard response
```
# ./api/tables/movie/ids/
{
   1: 'Star Wars',
   2: 'Return of the Jedi,
   ...
}
``` 
I don't know yet how to trigger Vue frontend based on the existence of the `column_name` query parameter. I don't know is this ok, or I completely miss the point.